extension/frontend/XliffEditor/main.js

```diff
@@ -2,21 +2,22 @@
  * This script will be run within the webview itself
  * It cannot access the main VS Code APIs directly.
  */
-
 (function () {
     const vscode = acquireVsCodeApi();
+    const oldState = vscode.getState();
 
+    if (oldState !== undefined) {
+        document.getElementById(oldState.position).focus();
```

Comment:
This does not happen for me... should it be done with a setTimeout(), so it'll give the renderer enough time?

Reply:
That could be an idea! Maybe based on number of transunits?

Reply:
maybe it's needed as well. But sometimes it's enough to just make the update async with setTimeout(). We'll see :)

Reply:
don't think it's a timing issue. `oldState` is always undefined here when I debug the webview... Don't know why, though...

Reply:
Even with the  AL test-app in the repo?

Reply:
Strange, the AL test-app seems to work... :/

Reply:
we need to check if `document.getElementById(oldState.position)` returns any element. The file might have changed so that it does not exist anymore. And when you are filtering, there's a big chance that it does not exist in your filtered set.